### ISSUE TITLE
Specs named with spaces have malformed stacks and no code frames for errors

### ISSUE THREAD

**Issue by chrisbreiding** [cypress-io/cypress#7553]
### Current behavior:

If a test fails or there is an error in a spec that has a space in its name, the following issues occur:

- The error stack does not point to the user code as it should
  - This also makes it impossible to open in IDE
- The stack line has `undefined:NaN` instead of `<line>:<column>` (_edit (7/9/20): only happens in `develop`_)
- The code frame does not display

![Screen Shot 2020-06-01 at 9 51 52 AM](https://user-images.githubusercontent.com/1157043/83415720-8b6d6380-a3ed-11ea-92fe-9fb08752342d.png)

### Desired behavior:

Errors should display properly regardless of file naming conventions.

### Test code to reproduce

- Create a spec file with a space in its name (e.g. `cypress/integration/with space_spec.js`)
- Make it fail

```javascript
it('fails', () => {
  expect('actual').to.equal('expected')
})
```

### Versions

Cypress 4.6.0+

**Comment by jennifer-shehane** [comment]
@chrisbreiding I'm not able to reproduce this exactly. The stack trace still isn't pointing to my spec file, but I don't see the undefined like you do. What am I missing 🤔 

### 4.5.0

<img width="684" alt="Screen Shot 2020-07-09 at 3 20 12 PM" src="https://user-images.githubusercontent.com/1271364/87018798-bff7ea80-c1f7-11ea-89c4-f2ba413775da.png">


### 4.6.0+

<img width="685" alt="Screen Shot 2020-07-09 at 3 18 15 PM" src="https://user-images.githubusercontent.com/1271364/87018820-c8e8bc00-c1f7-11ea-86b4-caec0bd533e2.png">

**Comment by chrisbreiding** [comment]
The difference is I was running the `develop` branch and not production Cypress. I suppose the `undefined` part wouldn't really be seen by users, but they'd see the other issues.

**Comment by bahmutov** [comment]
Released in v6.2.0
